packages/shared/types/bookmarks.ts

```diff
@@ -12,6 +12,12 @@ export const enum BookmarkTypes {
   UNKNOWN = "unknown",
 }
 
+export const zSortOrder = z.enum(["asc", "desc"]);
+export type SortOrder = z.infer<typeof zSortOrder>;
```

Comment:
Please follow the same style in the rest of the file. All zod inferred type start with the `Z` prefix.

Reply:
Done